sas9.4-viya/azuredeploy.json

```diff
@@ -0,0 +1,3683 @@
+        "SASDepot Folder": {
+           "type": "String",
+            "metadata": {
+                "description": "Folder Name in Azure File share where SAS94 depot is located"
+            }
+        },
+        "Viyarepo Folder": {
+            "type": "String",
+            "metadata": {
+                "description": "Folder Name in Azure File share where SAS Viya Repo is located"
+            }
+        },
+        "SAS Server license file": {
+            "type": "String",
+            "metadata": {
+                "description": "Name of SAS Application Server License file.You will find this file inside the SAS Software Depot. It should be inside the folder sid_file."
+            }
+        },
+        "KeyVault Owner ID": {
+            "type": "String",
+            "metadata": {
+                "description": "Key Vault Owner Object ID,Specifies the object ID of a user, service principal or security group in the Azure Active Directory tenant for the vault.Get it by using Get-AzADUser or Get-AzADServicePrincipal cmdlets. e.g.In Azure Cloud PowerShell type PS> Get-AzADUser -UserPrincipalName user@domain.com | grep Id"
+            }
+        },
+        "SSHPublicKey": {
+            "type": "String",
+            "metadata": {
+                "description": "The full ssh public key that will be added to the servers."
+            }
+        },
+        "Location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Azure Resources location, where all the SAS 94 and Viya resources should be created. e.g. servers, disks, IP's etc."
+            }
+        },
+        "_artifactsLocation": {
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/sas9.4-viya/",
+            "type": "String",
+            "metadata": {
+                "description": "For a standard deployment, keep the default.  The https URL to the base of the deployment files in Microsoft Azure. If a SAS key is needed, please do not include the SAS key in the URL. Instead, add the part after and include the question mark to the _artifactsLocationSasToken variable. ex. https://(STORAGE_ACCOUNT_FQDN)/(SHARE_NAME)/(SUBDIRECTORY_NAME)/"
+            }
+        },
+        "_artifactsLocationSasToken": {
+            "type": "securestring",
+            "defaultValue": "",
+            "metadata": {
+              "description": "For a standard deployment, leave empty. If you are running from a blob template, then provide the Shared Access Signature token (starting with a ?) that grants authorization to the private template. "
+            } 
+        }
+    },
+    "variables": {
+        "_artifactsLocationSasToken": "[parameters('_artifactsLocationSasToken')]",
+        "access_policy_template": "[concat(parameters('_artifactsLocation'),'nestedtemplates/access_policy.json', variables('_artifactsLocationSasToken'))]",

```

Comment:
use the uri() function for uris

Reply:
modified the syntax used uri() function.